src/Jank/TermManagement/Term.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { DeleteButton, TermText } from './ui';
+
+interface TermProps {
+  value: string
+}
+
+const Term = ({ value } : TermProps) => (
+  <li>
+    <TermText>{value}</TermText>
+    <DeleteButton>X</DeleteButton>
```

Comment:
This likely needs a `type="button"` prop, otherwise this will end up being a `type="submit"` button, which is the default behavior for `<button>` elements that don't define any `type`. As we don't submit any form here, it should be reflected in the type. Some browsers will even do an actual submit, causing the page to reload.
```suggestion
    <DeleteButton type="button">X</DeleteButton>
```

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

> The default behavior of the button. Possible values are:
>    *submit*: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a `<form>`, or if the attribute is an empty or invalid value.
>    *reset*: The button resets all the controls to their initial values, like `<input type="reset">`. (This behavior tends to annoy users.)
>    *button*: The button has no default behavior, and does nothing when pressed by default. It can have client-side scripts listen to the element's events, which are triggered when the events occur.

As we want to run client-side scripts (i.e. update our components without reloading the page), we should use `button` here. When using a form, the `submit` is fine - we can still intercept the `onSubmit` of the `form` in order to prevent a page reload.